src/Events/Event.ts

```diff
@@ -44,6 +45,7 @@ export class Event {
         this.categories = json.categories ? json.categories!.map((c: ChannelJson) => Category.fromJson(c)) : null
         this.name = json.name || null
         this.date = json.date ? LocalDate.parse(json.date) : null
+        this.isInThePast = json.isInThePast
```

Comment:
Is the JSON populated for all events, i.e. no need for a default value?

Reply:
@mortendevold It's represented as a boolean primitive in the core-v2 TO, so it will always be populated.